cmd/cape/cmd/delete.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/spf13/cobra"
+
+	"github.com/capeprivacy/cli/entities"
+)
+
+// deleteCmd represents the run command
+var deleteCmd = &cobra.Command{
+	Use:   "delete function_id",
+	Short: "Delete a deployed function",
+	Long:  "Delete a deployed function, takes function id.\n",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := delete(cmd, args)
+		if _, ok := err.(UserError); !ok {
+			cmd.SilenceUsage = true
+		}
+		return err
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(deleteCmd)
+}
+
+func delete(cmd *cobra.Command, args []string) error {
+	u := C.SupervisorHost
+	insecure := C.Insecure
+
+	if len(args) != 1 {
+		return UserError{Msg: "you must pass a function ID", Err: fmt.Errorf("invalid number of input arguments")}
+	}
+
+	functionID := args[0]
+
+	t, err := getAuthToken()
+	if err != nil {
+		return err
+	}
+	auth := entities.FunctionAuth{Type: entities.AuthenticationTypeAuth0, Token: t}
+	err = doDelete(u, functionID, insecure, auth)
+	if err != nil {
+		return fmt.Errorf("error processing data: %w", err)
```

Comment:
can we fix this error message?

Reply:
How about "error calling delete endpoint"?